src/main/scala/uk/gov/hmrc/hmrcfrontend/views/html/components/package.scala

```diff
@@ -82,4 +83,8 @@ package object components extends Utils with Aliases {
   @deprecated(message = "Use DI", since = "Play 2.6")
   lazy val HmrcCharacterCount = new hmrcCharacterCount(GovukTextarea, GovukHint)
 
+  type HmrcAddToAList = hmrcAddToAList
+  @deprecated(message = "Use DI", since = "Play 2.6")
+  lazy val HmrcAddToAList = Play.current.injector.instanceOf[HmrcAddToAList]
```

Comment:
This doesn't work, but I'm not quite sure what the important bit I've missed that's different from https://github.com/hmrc/play-frontend-hmrc/blob/master/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/html/deprecated/deprecatedPlay26Helpers.scala is 🤨 . @JoPintoPaul and @matthewmascord would either of you be able to help me understand what's gone wrong here? I'm unsure why this won't build on CI but does locally, I'm missing something crucial

Reply:
I understand the issue now that it's using deprecated functionality that isn't compatible with 2.8, not sure if there's any salvageable alternative here or I've hit a limitation I hadn't come across yet with these static components still around and I guess if so will need to change the implementation to still be able to support this

Reply:
Play.current isn't a think in 2.8. Can you instantiate with new as with the ones above?